test/FunctionalTests/Microsoft.OData.Core.Tests/Build.NetFramework/SelectExpandClauseExtensionsTests.cs

```diff
@@ -0,0 +1,138 @@
+﻿//---------------------------------------------------------------------
+// <copyright file="SelectExpandClauseExtensionsTests.cs" company="Microsoft">
+//      Copyright (C) Microsoft Corporation. All rights reserved. See License.txt in the project root for license information.
+// </copyright>
+//---------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.OData.Tests.UriParser;
+using Microsoft.OData.Tests.UriParser.Binders;
+using Microsoft.OData.UriParser;
+using Microsoft.OData.Edm;
+using Xunit;
+using ODataErrorStrings = Microsoft.OData.Strings;
+using Microsoft.OData.UriParser.Aggregation;
+
+namespace Microsoft.OData.Tests

```

Comment:
>Microsoft.OData.Tests [](start = 10, length = 21)

put this file into correctly folder and rename the namespace

Reply:
#Resolved